proto/pdpb.proto

```diff
@@ -558,3 +561,15 @@ message GetOperatorResponse {
     OperatorStatus status = 4;
     bytes kind = 5;
 }
+
+message SetGCSafePointLimitRequest {
+    RequestHeader header = 1;
+    // service_name/uuid_v4
+    bytes id = 2;
```

Comment:
How about adding the `id` and `ttl` field in the `UpdateGCSafepoint` method, then it is not necessary to add a new method.